damus/Views/Images/ProfilePicImageView.swift

```diff
@@ -93,18 +96,57 @@ struct ProfilePicImageView: View {
                 .ignoresSafeArea()
             
             ZoomableScrollView {
-                ProfileImageContainerView(url: get_profile_url(picture: nil, pubkey: pubkey, profiles: profiles), settings: settings)
+                ProfileImageContainerView(url: get_profile_url(picture: nil, pubkey: pubkey, profiles: profiles), settings: settings, image: $image)
                     .aspectRatio(contentMode: .fit)
                     .padding(.top, Theme.safeAreaInsets?.top)
                     .padding(.bottom, Theme.safeAreaInsets?.bottom)
                     .padding(.horizontal)
+                    .allowsHitTesting(false)
             }
             .ignoresSafeArea()
             .modifier(SwipeToDismissModifier(minDistance: 50, onDismiss: {
                 presentationMode.wrappedValue.dismiss()
             }))
         }
-        .overlay(NavDismissBarView(navDismissBarContainer: .profilePicImageView), alignment: .top)
+        .overlay(
+            Group {
+                if showMenu {
+                    HStack {
+                        NavDismissBarView()
```

Comment:
This line is failing to compile with the following error message: 

```
Missing argument for parameter 'navDismissBarContainer' in call
```

I believe it is due to a recent parallel change on `NavDismissBarView`, that now requires an argument for this call